bokeh/plotting/helpers.py

```diff
@@ -491,7 +525,8 @@ def func(self, *args, **kwargs):
     kwlines = []
     kws = glyphclass.properties() - set(glyphclass._args)
     for kw in sorted(kws):
-        if kw == "session": continue # TODO (bev) improve or remove
+        if kw == "session":
+            continue  # TODO (bev) improve or remove
```

Comment:
ok, i was just pep8-ing, this has nothing to do with this code. i'm not going to remove some code in this PR. Happy to do it in a follow-on though.